servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/RoundRobinLoadBalancerFactory.java

```diff
@@ -140,24 +144,18 @@ public ExecutionStrategy requiredOffloads() {
 
         /**
          * Creates a new instance with default settings.
+         *
+         * @deprecated use {@link RoundRobinLoadBalancers#builder(String)} instead.
          */
+        @Deprecated // FIXME: 0.43 - remove deprecated constructor
         public Builder() {
+            this(UUID.randomUUID().toString());
+        }
+
+        Builder(final String id) {
+            this.id = id;
```

Comment:
Either here or in `RoundRobingLoadBalancers`, consider adding a check that the `id` is not empty